modules/ROOT/pages/clustering/databases.adoc

```diff
@@ -36,6 +36,23 @@ However, over time, or after several `CREATE DATABASE` commands have been issued
 At this point you can run `REALLOCATE DATABASES` to make the cluster re-balance databases across all servers that are part of the cluster.
 ====
 
+The `CREATE DATABASE` and `ALTER DATABASE` commands support the usage of link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/syntax/parameters[parameters] when providing the number of primaries and secondaries.
```

Comment:
I wonder if this is overkill. We don't mention that the name could be a parameter. Maybe enough to add a second example to the create example. Something like:

```cypher
CREATE DATABASE foo TOPOLOGY 3 PRIMARIES 2 SECONDARIES
```

or with parameters

```cypher
{
  "dbname": "foo",
  "primary": 1,
  "secondary": 2
}

CREATE DATABASE $dbName TOPOLOGY $primary PRIMARIES $secondary SECONDARIES

Reply:
I think it's useful to have both.

Reply:
Added the dbname 👍 

I feel like it would be overkill to have one parameter example for create and another for alter alter, by mentioning ALTER in the text we get away with only the one. Let me know if you'd prefer the `or with parameters` approach with one for create and one for alter 🤷

Reply:
I think a better practice is to keep them separate because whoever is looking for how to alter a database won't be looking at the create a database section. Of course, they might find a way to the example, but nobody reads the docs from top to bottom. A workaround could be to mention the feature, from when it is supported, etc., in the alter section and then refer to the create section, but it might be even more complicated.